data/gui/themes/default/window/game_version.cfg

```diff
@@ -258,6 +258,39 @@
 						[/grid]
 					[/column]
 				[/row]
+				[row]
+					[column]
+						border = "left,bottom"
+						border_size = 5
+						horizontal_alignment = left
+						[grid]
+							[row]
+								[column]
+									horizontal_alignment = left
+									[label]
+										label = _ "<span font_family='DejaVu Sans'>✦</span> <b>Manual:</b>"
+										use_markup = yes
+									[/label]
+								[/column]
+								[column]
+									grow_factor = 0
+									horizontal_alignment = left
+									border = "left"
+									border_size = 5
+									
+									[label]
+										#textdomain wesnoth-test
+                                        #po: Replace with your local version of the Wesnoth Manual found at https://wiki.wesnoth.org/WesnothManual
+										label = _ "<span foreground='yellow'><u>https://www.wesnoth.org/manual/dev/manual.en.html</u></span>"
```

Comment:
Instead of requiring manual translation this URL should just be generated based on the selected translation.